packages/features/bookings/lib/getBookingFields.ts

```diff
@@ -282,6 +274,18 @@ export const ensureBookingInputsHaveSystemFields = ({
 
   bookingFields = missingSystemBeforeFields.concat(bookingFields);
 
+  // Backward Compatibility for SMS Reminder Number
+  // Note: We still need workflows in `getBookingFields` due to Backward Compatibility. If we do a one time entry for all event-types, we can remove workflows from `getBookingFields`
+  // Also, note that even if Workflows don't explicity add smsReminderNumber field to bookingFields, it would be added as a side effect of this backward compatibility logic
```

Comment:
@CarinaWolli Also wanted to bring to your notice that, even if we remove all that CRUD logic for `smsReminderNumber` field from workflows router, the field will still be shown due to this logic.

This logic is supposed to be temporary. The intention is to remove the requirement of workflows in bookingFields logic so that it can stay fast.

But I can't deny the fact that it keeps things simple.